models/preference_map_persister.go

```diff
@@ -29,50 +28,49 @@ func (s *MapPreferencePersister) ReadFromPersister(userID string) (*Preference,
 	}
 
 	if s.db == nil {
-		return nil, errors.New("connection to DB does not exist")
+		return nil, ErrDBConnection
 	}
 
 	if userID == "" {
-		return nil, errors.New("user ID is empty")
+		return nil, ErrUserID
 	}
 
 	dataCopyB, ok := s.db.Load(userID)
 	if ok {
-		logrus.Debugf("retrieved session for user with id: %s", userID)
+		log.Debugf("retrieved session for user with id: %s", userID)
 		newData, ok1 := dataCopyB.(*Preference)
 		if ok1 {
-			logrus.Debugf("session for user with id: %s was read in tact.", userID)
+			log.Debugf("session for user with id: %s was read in tact.", userID)
 			data = newData
 		} else {
-			logrus.Warnf("session for user with id: %s was NOT read in tact.", userID)
+			log.Warnf("session for user with id: %s was NOT read in tact.", userID)
 		}
 	} else {
-		logrus.Warnf("unable to find session for user with id: %s.", userID)
+		log.Warnf("unable to find session for user with id: %s.", userID)
 	}
 	return data, nil
 }
 
 // WriteToPersister persists session for the user
 func (s *MapPreferencePersister) WriteToPersister(userID string, data *Preference) error {
 	if s.db == nil {
-		return errors.New("connection to DB does not exist")
+		return ErrDBConnection
 	}
 
 	if userID == "" {
-		return errors.New("user ID is empty")
```

Comment:
This text is helpful. Where is it now being captured?

Reply:
Hey @leecalcote the variable `ErrUserID` has captured this text now. In error.go this `ErrUserID` is present. `	ErrUserID               = errors.New(ErrUserIDCode, errors.Alert, []string{"User ID is empty"}, []string{}, []string{}, []string{})
`